attestation.js

```diff
@@ -6,18 +6,46 @@ const db = require('byteballcore/db');
 const eventBus = require('byteballcore/event_bus');
 const validationUtils = require('byteballcore/validation_utils');
 const mail = require('byteballcore/mail');
+const emailjs = require('emailjs');
 const texts = require('./modules/texts');
 const reward = require('./modules/reward');
 const conversion = require('./modules/conversion');
 const emailAttestation = require('./modules/email_attestation');
 const notifications = require('./modules/notifications');
 const randomCryptoString = require('./modules/random-crypto-string');
+const i18nModule = require("i18n");
+const arrWhitelistEmails = Object.keys(conf.objRewardWhiteListEmails);
+let server;
+
+if (conf.bUseSmtp) {
+	server = emailjs.server.connect({
+		user: conf.smtpUser,
+		password: conf.smtpPassword,
+		host: conf.smtpHost,
+		port: typeof conf.smtpPort == 'undefined'? 465 : conf.smtpPort,
+		ssl: typeof conf.smtpSsl == 'undefined'? true : conf.smtpSsl
+	});
+}
+
+var arrLanguages = [];
+if (conf.isMultiLingual) {
+	for (var index in conf.languagesAvailable) {
+		arrLanguages.push(conf.languagesAvailable[index].file);
+	}
+}
+
+i18nModule.configure({
+	locales: arrLanguages,
+	directory: __dirname + '/locales'
+});
+var i18n = {};
+i18nModule.init(i18n);
 
 /**
  * user pairs his device with bot
  */
 eventBus.on('paired', (from_address) => {
-	respond(from_address, '', texts.greeting());
+	respond(from_address, '');
```

Comment:
empty greeting?

Reply:
Good catch, fixed it. Was a problem only if conf.isMultiLingual is false.

Reply:
Moved greeting back to Respond function because I don't have a user's selected language at this point in case user has been already paired once, removes the bot and pairs again.